lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -59,6 +61,7 @@
       - running
       - restarted
       - stopped
+      - terminate
```

Comment:
terminated. Also it is not added in SCALEWAY_SERVER_STATES neither in SCALEWAY_TRANSITIONS_STATES (terminating)?